docs/modules/ROOT/pages/index.adoc

```diff
@@ -289,6 +291,7 @@ An example in https://www.rfc-editor.org/rfc/rfc8949.html#name-diagnostic-notati
 ----
 {
   "signer_payload": {
+    "sig_type": "cawg.x509.cose",
```

Comment:
I assume that's ok. Where are the possible values are defined?

Reply:
I see they are defined later in the doc... Ignore my previous comment (or perhaps add a ref to where the valid values are defined)

Reply:
todo: remove sig-type from the cddl of the outer map